src/Microsoft.Health.Fhir.Api/Features/Routing/KnownRoutes.cs

```diff
@@ -58,6 +58,9 @@ internal class KnownRoutes
         public const string Everything = "$everything";
         public const string PatientEverythingById = "Patient/" + IdRouteSegment + "/" + Everything;
 
+        public const string Purge = "$purge";
```

Comment:
>$purge [](http://example.com/codeflow?start=37&length=6)

`PurgeHistory` maybe?
I would expect recent version to be deleted from purge as well.

Reply:
I really like this suggestion :) PurgeHistory makes more sense to me too.